hera_sim/tests/test_interpolators.py

```diff
@@ -0,0 +1,59 @@
+# let's figure out some tests
+
+import numpy as np
+import os
+import glob
+import tempfile
+import shutil
+
+from scipy.interpolate import RectBivariateSpline
+from hera_sim.interpolators import Tsky
+from hera_sim.data import DATA_PATH
+from nose.tools import raises
+
+def test_interpolator():
+    # make a temporary file
```

Comment:
You don't seem to be actually using the tempfile module in this function?

Reply:
Nope; I only need a temporary `.npz` file for this test. So I thought it would be best to generate some data in the test, save it to a `.npz`, use the `.npz` to instantiate a `Tsky` object, then delete the `.npz` when I'm done with it.

Reply:
You can definitely save `.npz` files to the temp dir though?

Reply:
Did you end up doing the `.npz` in a temp file? I'm fine with either way, so I guess it doesn't matter :-)

Reply:
I made a temporary directory with `tempfile.mkdtemp` and saved the `.npz` file there.